src/components/AttendanceChangeModal.tsx

```diff
@@ -22,6 +22,14 @@ const AttendanceChangeModal = ({
     null
   );
 
+  const isDepatureTimeDisabled =
```

Comment:
You could make the variables the inverse of these: `const departureTimeEnabled` and then use the inverse when checking, so you can check to disable if enabled == False.